pupil/views/partials/_question_content.ejs

```diff
@@ -1,3 +1,3 @@
-<main id="content" class="js-content" role="main">
+<main id="content" class="js-content">
```

Comment:
Why remove `role="main"`?  Most gov.uk web pages have this.  I assume it tested better.